powersimdata/input/change_table.py

```diff
@@ -646,6 +673,11 @@ def add_plant(self, info):
 
         :param list info: each entry is a dictionary. The dictionary gathers
             the information needed to create a new generator.
+            Required keys: "bus_id", "Pmax", "type".
+            Optional keys: "c0", "c1", "c2", "Pmin".
+            "c0", "c1", and "c2" are the coefficients for the cost curve, representing
+            the fixed cost ($/hour), linear cost ($/MWh), and quadratic cost ($/MW^2*h).
+            These are optional for hydro, solar, and wind, and required for other types.
         :raises TypeError: if info is not a list.
```

Comment:
info --> \`\`info\`\`

Reply:
With one ``` or two on each side?

Reply:
two on each side for verbatim (it displays like `that`) in ReST. I started to reference to parameters in the docstring this way as we do in Slack or Markdown

Reply:
@rouille Good to know.

Reply:
Done.